tests/test_bot.py

```
@@ -561,3 +561,202 @@ async def my_task(self):
+    e1 = MagicMock(spec=discord.Emoji)
+    e2 = MagicMock(spec=discord.Emoji)
+    guild.fetch_emojis = AsyncMock(return_value=[e1, e2])
+    bot.get_guild = MagicMock(return_value=guild)
+    result = await bot.fetch_guild_emojis(123)
+    assert result == [e1, e2]
+
+
+# ── @bot.on_error ─────────────────────────────────────────────────────────────
+
+async def test_on_error_registers_handler(bot):
+    async def handler(ctx, error):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `handler`

Add return type annotation: `None`

(ANN202)